src/main/java/uk/gov/hmcts/reform/sandl/snlevents/Application.java

```diff
@@ -3,11 +3,9 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker;
-import org.springframework.cloud.netflix.hystrix.dashboard.EnableHystrixDashboard;
 
 @SpringBootApplication
 @EnableCircuitBreaker
-@EnableHystrixDashboard
```

Comment:
Hystrix dashboard seems not to work with our springboot version. We can consider using netflix turbine, but it would be another service to deploy just to watch hystrix metrics from 2 services. As our application is not even used at this moment I would postpone introducing of turbine.